src/modal_for_noobs/auth_manager.py

```
@@ -259,6 +260,217 @@ def open_tokens_page(self) -> None:
+            }
+            
+    def get_huggingface_auth_status(self) -> Dict[str, any]:
+        """Get Hugging Face authentication status.
+        
+        Returns:
+            Dictionary with authentication status information
+        """
+        try:
+            hf_auth_file = self.config_dir / "huggingface_auth.json"
+            if hf_auth_file.exists():
+                with open(hf_auth_file, "r") as f:
```

Ruff (0.11.9)

`open()` should be replaced by `Path.open()`

(PTH123)